package.json

```diff
@@ -53,7 +53,7 @@
     "serve-favicon": "^2.4.5",
     "source-map-support": "^0.5.5",
     "validate.js": "^0.13.0",
-    "webdriverio": "^5.10.9",
+    "webdriverio": "^6.0.2",
```

Comment:
Why is this bumped in _this_ PR? Seems unrelated.

Reply:
rebase glitch. Will revert